app/router.js

```diff
@@ -53,7 +53,9 @@ router.map(function() {
         this.route('attendees');
         this.route('add-order');
         this.route('discount-codes');
-        this.route('access', { path: 'access-codes' });
+        this.route('access', { path: 'access-codes' }, function() {
```

Comment:
Call the route itself as `access-codes` instead of overriding the path like this